src/utils.jl

```diff
@@ -29,21 +29,26 @@ function get_symbols_in_module(m::Module)
 end
 
 """
-    call_with_catch( juliafunc, arguments )
+    call_with_catch(juliafunc, arguments)
 
-Return a tuple `( ok, val )`
+Return a tuple `(ok, val)`
 where `ok` is either `true`, meaning that calling the function `juliafunc`
 with `arguments` returns the value `val`,
 or `false`, meaning that the function call runs into an error;
 in the latter case, `val` is set to the string of the error message.
 
 # Examples
 ```jldoctest
-julia> GAP.call_with_catch( sqrt, 2 )
+julia> GAP.call_with_catch(sqrt, 2)
 (true, 1.4142135623730951)
 
-julia> GAP.call_with_catch( sqrt, -2 )
-(false, "DomainError(-2.0, \\"sqrt will only return a complex result if called with a complex argument. Try sqrt(Complex(x)).\\")")
```

Comment:
It seems there is a similar problem (changes in the error in Julia nightly) inside `pkg/JuliaInterface/tst/utils.tst`